Octokit/Models/Response/UserType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Octokit
+{
+    public enum UserType
```

Comment:
Sorry, I should have been clearer here:

 - [This link](https://help.github.com/articles/searching-users/#search-for-users-or-organizations) indicates when searching based on user you use `type:org` in the request parameters.
 - [This link](https://developer.github.com/v3/orgs/#get-an-organization) shows the response you get back has `"type": "Organization"` in the response.